tests/constants/admincli/cmd.go

```diff
@@ -79,4 +79,15 @@ const (
 
 	//RemoveVolumes option refers to removing all volumes from a vmgroup
 	RemoveVolumes = " --remove-volumes"
+
+	// CreateVsanPolicy referring to create vsanPolicy
+	// where --name will be name of the vsanPolicy
+	CreateVsanPolicy = vmdkopsAdmin + "policy create --name="
```

Comment:
I think we already have a constant to create policy in the same file. Not sure if this is needed.